lite2/client.go

```diff
@@ -965,13 +965,19 @@ func (c *Client) RemoveNoLongerTrustedHeaders(now time.Time) {
 		c.logger.Error("can't get first trusted height", "err", err)
 		return
 	}
+	if oldestHeight == -1 { // no headers yet => wait
+		return
+	}
 
 	// 2) Get the latest height.
 	latestHeight, err := c.LastTrustedHeight()
 	if err != nil {
 		c.logger.Error("can't get last trusted height", "err", err)
 		return
 	}
+	if latestHeight == -1 { // no headers yet => wait
+		return
+	}
 
```

Comment:
Are both if statements necessary? Is there a circumstance where if there are no headers that asking for the first header doesn't return -1

Reply:
no, but it's okay to have both I think